diff --git i/src/Makefile.am w/src/Makefile.am
index fd1d73c..9f5bf81 100644
--- i/src/Makefile.am
+++ w/src/Makefile.am
@@ -20,6 +20,7 @@ libpsl_la_LDFLAGS = -no-undefined -version-info $(LIBPSL_SO_VERSION)
 # PSL_FILE can be set by ./configure --with-psl-file=[PATH]
 suffixes_dafsa.h: $(PSL_FILE) $(srcdir)/psl-make-dafsa
 	$(PYTHON) $(srcdir)/psl-make-dafsa --output-format=cxx+ "$(PSL_FILE)" suffixes_dafsa.h
+	chtag -tc 819 suffixes_dafsa.h
 
 bin_SCRIPTS = psl-make-dafsa
 
